pkg/repository/prisma/dbsqlc/queue.sql

```diff
@@ -633,7 +633,7 @@ WITH retries AS (
         srs
```

Comment:
@abelanger5 i'm a bit stumped here, should we be processing `updated_step_runs` and `inserted_sqs` serially in `processStepRunUpdatesV2`?

I'm